tests/test_cameras.py

```diff
@@ -78,24 +109,24 @@ def test_camera(request, robot_type):
         camera.read()
     color_image = camera.read()
     async_color_image = camera.async_read()
-    print(
+    error_msg = (
         "max_pixel_difference between read() and async_read()",
         compute_max_pixel_difference(color_image, async_color_image),
     )
-    assert np.allclose(color_image, async_color_image, rtol=1e-5, atol=MAX_PIXEL_DIFFERENCE)
+    assert np.allclose(color_image, async_color_image, rtol=1e-5, atol=MAX_PIXEL_DIFFERENCE), error_msg
 
     # Test disconnecting
     camera.disconnect()
     assert camera.camera is None
     assert camera.thread is None
 
     # Test disconnecting with `__del__`
-    camera = OpenCVCamera(CAMERA_INDEX)
+    camera = make_camera(camera_type)
     camera.connect()
     del camera
```

Comment:
Nit: What exactly is this testing and how is it being tested? I would expect an assertion perhaps something like 


```

try:
    camera  # Attempt to access the deleted variable
    assert False, "Expected a NameError but camera still exists"  # If camera still exists, raise an assertion failure
except NameError:
    assert True  # The NameError is raised as expected, so the assert passes```

Reply:
I think it's ok, we are just making sure we can instantiate a camera, connect it and delete it. 
We dont expect any error.